### ISSUE TITLE
Version 0.6.2 can not response events when widget in the header, such as Inkwell、Button etc.

### ISSUE THREAD

**Issue by realWayneZhang** [letsar/flutter_sticky_header#75]
You can duplicate the issue by running the code as below.

```
import 'package:flutter/material.dart';
import 'package:flutter_sticky_header/flutter_sticky_header.dart';

class TabHeaderExample extends StatefulWidget {
  const TabHeaderExample({Key? key}) : super(key: key);

  @override
  State<TabHeaderExample> createState() => _TabHeaderExampleState();
}

class _TabHeaderExampleState extends State<TabHeaderExample> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: Text('Tab List Example'),),
      body: CustomScrollView(
        slivers: [
          _StickyHeaderList(index: 0),
          _StickyHeaderList(index: 1),
          _StickyHeaderList(index: 2),
          _StickyHeaderList(index: 3),
        ],
      ),
    );
  }
}

class _StickyHeaderList extends StatefulWidget  {
  final int index;
  const _StickyHeaderList({Key? key, required this.index}) : super(key: key);

  @override
  State<_StickyHeaderList> createState() => _StickyHeaderListState();
}

class _StickyHeaderListState extends State<_StickyHeaderList> with TickerProviderStateMixin {

  late TabController _tabController;

  @override
  void initState() {
    super.initState();
    _tabController = TabController(length: 10, vsync: this);
  }

  @override
  Widget build(BuildContext context) {
    return SliverStickyHeader(
      header: Container(
        color: Colors.white,
        height: 47,
        child: Row(
          mainAxisAlignment: MainAxisAlignment.spaceBetween,
          crossAxisAlignment: CrossAxisAlignment.center,
          children: [
            Expanded(
              child: TabBar(
                isScrollable: true,
                controller: _tabController,
                indicatorColor: const Color(0xFF216DDF),
                indicatorSize: TabBarIndicatorSize.label,
                indicatorWeight: 2,
                indicatorPadding: EdgeInsets.only(bottom: 10),
                unselectedLabelColor: const Color(0xFF888888),
                unselectedLabelStyle: TextStyle(fontSize: 17),
                labelColor: const Color(0xFF216DDF),
                labelStyle: TextStyle(fontSize: 17, fontWeight: FontWeight.bold),
                tabs: _createTabs(),
                onTap: (index) { print(index);},
              ),
            ),
            InkWell(
              child: Container(
                decoration: BoxDecoration(
                  color: Colors.red,
                  boxShadow: [
                    BoxShadow(
                      color: Colors.white,
                      blurRadius: 8,
                      offset: Offset(-20, 0),
                    ),
                  ],
                ),
                padding: EdgeInsets.only(left: 5, right: 12),
                child: Row(
                  children: [
                    Text(
                      "全部",
                      style: TextStyle(color: const Color(0xFF101010), fontSize: 17),
                    ),
                  ],
                ),
              ),
              onTap: () {
                // Can not response
                print("1234567");
              },
            ),
          ],
        ),
      ),
      sliver: SliverList(
        delegate: SliverChildBuilderDelegate(
              (context, i) => ListTile(
            onTap: () {
              print('tile $i');
            },
            leading: CircleAvatar(
              child: Text('${widget.index}'),
            ),
            title: Text('List tile #$i'),
          ),
          childCount: 6,
        ),
      ),
    );
  }

  /// 创建Tabs
  List<Widget> _createTabs() {
    return ["我是tab", "我是tab","我是tab","我是tab","我是tab","我是tab","我是tab","我是tab","我是tab","我是tab"].map((e) => Tab(text: e)).toList();
  }
}
```

**Comment by abellee** [comment]
me too

**Comment by HamzaMukhtar1** [comment]
I'm getting this is too

**Comment by xuhongfei** [comment]
same issue

**Comment by helloV7** [comment]
`pubspec.yaml `  file 
`flutter_sticky_header: ^0.6.1 `to` flutter_sticky_header: 0.6.1` 
  then run ` flutter pub get`

**Comment by KanybekMomukeyev** [comment]
2 day researched this issue, and finally find the reason

**Comment by amsayk** [comment]
@letsar 

This issue still exists when the sticky header is in a snapped state. It becomes impossible to click on anything rendered with the header.

Thanks for amazing library!

**Comment by letsar** [comment]
@amsayk have you a complete example I could run? I didn't reproduced what you described

**Comment by amsayk** [comment]
@letsar Here is a `main.dart` to reproduce it. The only package installed is `flutter_sticky_header` version `0.6.3`.

It works if you comment out the `CupertinoSliverNavigationBar `. Otherwise, no other factor seems to matter.

```dart

import 'package:flutter/material.dart';
import 'package:flutter/cupertino.dart';

import 'package:flutter_sticky_header/flutter_sticky_header.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return const CupertinoApp(
      debugShowCheckedModeBanner: false,
      title: 'Flutter Demo',
      home: Scaffold(
        body: MyHomePage(title: 'Flutter Demo Home Page'),
      ),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({Key? key, required this.title}) : super(key: key);

  // This widget is the home page of your application. It is stateful, meaning
  // that it has a State object (defined below) that contains fields that affect
  // how it looks.

  // This class is the configuration for the state. It holds the values (in this
  // case the title) provided by the parent (in this case the App widget) and
  // used by the build method of the State. Fields in a Widget subclass are
  // always marked "final".

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  @override
  Widget build(BuildContext context) {
    return CupertinoPageScaffold(
      child: CustomScrollView(
        primary: true,
        physics: const AlwaysScrollableScrollPhysics(parent: ClampingScrollPhysics()),
        slivers: [
          CupertinoSliverNavigationBar(
            key: const ValueKey('NavBar'),
            previousPageTitle: 'Back',
            border: Border(
              bottom: BorderSide(
                color: Theme.of(context).dividerColor,
                width: 1.0, // 0.0 means one physical pixel
              ),
            ),
            stretch: true,
            largeTitle: const Text('Details'),
          ),
          SliverStickyHeader.builder(
            builder: (context, state) => ListTile(
              tileColor: Colors.grey,
              shape: RoundedRectangleBorder(side: const BorderSide(color: Colors.black, width: 1), borderRadius: BorderRadius.circular(0)),
              leading: const CircleAvatar(),
              title: const Text('Title 1'),
              trailing: MouseRegion(
                cursor: SystemMouseCursors.click,
                child: GestureDetector(
                  child: Icon(
                    Icons.settings,
                    size: 28.0,
                    color: Theme.of(context).primaryColor,
                  ),
                  onTapDown: (details) => print('onTapDown: $details'),
                ),
              ),
            ),
            sliver: SliverList(
              delegate: SliverChildListDelegate(
                List.generate(
                  10,
                  (index) => Container(
                    child: SizedBox(
                      height: 250.0,
                      child: Center(child: Text('Element $index')),
                    ),
                  ),
                ),
              ),
            ),
          ),
          SliverStickyHeader.builder(
            builder: (context, state) => ListTile(
              tileColor: Colors.grey,
              shape: RoundedRectangleBorder(side: const BorderSide(color: Colors.black, width: 1), borderRadius: BorderRadius.circular(0)),
              leading: const CircleAvatar(),
              title: const Text('Title 2'),
              trailing: MouseRegion(
                cursor: SystemMouseCursors.click,
                child: GestureDetector(
                  child: Icon(
                    Icons.settings,
                    size: 28.0,
                    color: Theme.of(context).primaryColor,
                  ),
                  onTapDown: (details) => print('onTapDown: $details'),
                ),
              ),
            ),
            sliver: SliverList(
              delegate: SliverChildListDelegate(
                List.generate(
                  10,
                  (index) => Container(
                    child: SizedBox(
                      height: 250.0,
                      child: Center(child: Text('Element $index')),
                    ),
                  ),
                ),
              ),
            ),
          ),
          SliverStickyHeader.builder(
            builder: (context, state) => ListTile(
              tileColor: Colors.grey,
              shape: RoundedRectangleBorder(side: const BorderSide(color: Colors.black, width: 1), borderRadius: BorderRadius.circular(0)),
              leading: const CircleAvatar(),
              title: const Text('Title 3'),
              trailing: MouseRegion(
                cursor: SystemMouseCursors.click,
                child: GestureDetector(
                  child: Icon(
                    Icons.settings,
                    size: 28.0,
                    color: Theme.of(context).primaryColor,
                  ),
                  onTapDown: (details) => print('onTapDown: $details'),
                ),
              ),
            ),
            sliver: SliverList(
              delegate: SliverChildListDelegate(
                List.generate(
                  10,
                  (index) => Container(
                    child: SizedBox(
                      height: 250.0,
                      child: Center(child: Text('Element $index')),
                    ),
                  ),
                ),
              ),
            ),
          ),
        ],
      ),
    );
  }
}


```

Hope it helps!

**Comment by mherbert93** [comment]
Experiencing the same issue with versions above 0.6.1.

**Comment by Tony0205** [comment]
I faced the same issue. This issue seems to have been resolved in version `flutter_sticky_header: ^0.6.3`.
and it works fine with version `flutter_sticky_header: 0.6.1` too.
